Training_node/first_project/app/controllers/users.js

```diff
@@ -103,3 +103,31 @@ exports.create = (req, res, next) => {
     next(errors.defaultError(validation.reason));
   }
 };
+
+exports.loggedUser = (req, res, next) => {
+  if (req.query.limit && req.query.page) {
+    const limitInt = parseInt(req.query.limit);
+    const pageInt = parseInt(req.query.page);
+    if (pageInt === 1) {
```

Comment:
do you need this?

Reply:
Yes, because req.query.limit and req.query.page are strings and i can't compare with ints.